sdk/typescript/src/rpc/websocket-client.ts

```diff
@@ -13,7 +13,14 @@ import { Client as WsRpcClient } from 'rpc-websockets';
 export const getWebsocketUrl = (httpUrl: string, port?: number): string => {
   const url = new URL(httpUrl);
   url.protocol = url.protocol.replace('http', 'ws');
-  url.port = (port ?? 9001).toString();
+  if (!url.port) {
```

Comment:
URL ignores the default ports, for example doing something like
```
const url = new URL('http://example.com:80')
console.log(url.port); // ''
```
port will be empty same for https. So this will always throw if we use default port without passing the port parameter.

I don't think we should throw errors if port is not set. If not set it will mean the default port for the scheme (http/ws -> 80, https/wss -> 443)